DP#195-239 Create Order Information Page

YuriiShp opened the pull request:
1.Added module oct/pages/models/order_info_page.py
2.Added import shortcut for the class OrderInfoPage to the pages package
3.Black formating

YuriiShp committed:
DP#195-239 Create Order Information Page

1.Added module oct/pages/models/order_info_page.py
2.Added import shortcut for the class OrderInfoPage to the pages package
3.Black formating

moiseienko-dg commented on oct/pages/models/order_info_page.py:
```diff
@@ -0,0 +1,120 @@
+
+    url = "index.php?route=account/order/info&order_id="
+
+    def __init__(self, order_id: int, **kwargs):
```
do we need **kwargs?

moiseienko-dg commented on oct/pages/models/order_info_page.py:
```diff
@@ -0,0 +1,120 @@
+    url = "index.php?route=account/order/info&order_id="
+
+    def __init__(self, order_id: int, **kwargs):
+        super().__init__(**kwargs)
```
the same

SerhiiKhyzhko committed:
Accessory page

Success page renamed to Accessory page

added to __init__ url var with defoaut 'success' meaning

changed import Success to Accessory in __init__.py

YuriiShp committed:
premerge

YuriiShp committed:
DP#195_269_Create_Search_Page

1.Added module oct/pages/models/search_page.py
2.Added Paginator class to the module oct/pages/base/elements.py
3.Added import shortcut for the class SearchPage to the pages package

YuriiShp committed:
?

YuriiShp committed:
Merge branch 'DP195_232_Update_Success_page' of https://github.com/ITA-Dnipro/dp-195-taqc-python into test

YuriiShp committed:
corrected OrderInfoPage  class constructor
